gems/manageiq-providers-amazon/config/initializers/seed.rb

```diff
@@ -0,0 +1 @@
+EvmDatabase::RAILS_ENGINE_MODEL_CLASS_NAMES << 'ManageIQ::Providers::Amazon::AeDatastore'
```

Comment:
@Fryguy @gmcculloug this is a bit hacky to add `ManageIQ::Providers::Amazon::AeDatastore` to the list of seedable classes.

Maybe I touch `EvmDatabase` to support a proper api for manipulation? Like I did with [DescendableLoader](https://github.com/ManageIQ/manageiq/blob/master/gems/manageiq-providers-amazon/config/initializers/descendant_loader.rb)

And I also don't really like doing it in an intializer. I think it should be in `gems/amazon/config/application.rb` ?